components/frontend/src/report/ReportsTitle.js

```diff
@@ -31,10 +32,22 @@ export function ReportsTitle(props) {
                         <MultipleChoiceInput
                             allowAdditions
                             label="Users allowed to edit reports (user name or email address)"
-                            options={props.editors}
+                            options={props.permissions.edit_reports || []}
                             placeholder="All authenticated users"
-                            set_value={(value) => set_reports_attribute("editors", value, props.reload)}
-                            value={props.editors}
+                            set_value={(value) => set_reports_attribute("permissions", {edit_reports: value}, props.reload)}
+                            value={props.permissions.edit_reports || []}
+                        />
+                    </Grid.Column>
+                </Grid.Row>
+                <Grid.Row columns={1}>
+                    <Grid.Column>
+                        <MultipleChoiceInput
+                            allowAdditions
+                            label="Users allowed to edit measured entities (user name or email address)"
```

Comment:
I don't think we use the term "entity" in the user interface at all, and I'm not sure we should start using it because it is such an abstract word. Maybe say: "Users allowed to manage false positives"?

Reply:
Let's just bite the bullet and introduce the term here, and add it to the user docs as well?

Reply:
Yeah, i'd say that would be a good idea.